manifests/base/code-server-notebook-imagestream.yaml

```diff
@@ -24,7 +24,7 @@ spec:
       from:
         kind: DockerImage
         name: $(odh-codeserver-notebook-image-n)
-      name: "2024.1"
+      name: "2024.2"
```

Comment:
Great Diamond! As part of this workload we will need to update as well the following annotations: `opendatahub.io/notebook-software` and  `opendatahub.io/notebook-python-dependencies` with the correct versions that we will provide on 2024.2 image streams. You can find the correct versions on each image Pipfile for example for datascience is:  https://github.com/opendatahub-io/notebooks/blob/main/jupyter/datascience/ubi9-python-3.11/Pipfile